CHANGELOG.md

```diff
@@ -15,7 +15,7 @@
 ### Misc. Updates
 
 - Updated **AWSMobileAnalytics** podspec with `deprecated` and `deprecated_in_favor_of` attributes
-
+- Added workaround for `use_modular_headers!` inside of Podfile (experimental)
```

Comment:
do we need documentation updates for this?

Reply:
Are you suggesting that we need to advertise support for modular headers in our documents(or something else?) ?  Using modular headers is a feature provided by  CocoaPods, and I think it is up to the user if they want to use it or not.